modules/apps/journal/journal-service/src/main/java/com/liferay/journal/service/impl/JournalArticleLocalServiceImpl.java

```diff
@@ -7432,6 +7432,12 @@ private Map<Locale, String> _checkFriendlyURLMap(
 				friendlyURLMap.entrySet()) {
 
 			if (Validator.isNotNull(friendlyURL.getValue())) {
+				if (Validator.isNull(friendlyURLMap.get(defaultLocale))) {
+					friendlyURLMap.replace(
+						defaultLocale,
+						_removeTrailingSlashes(titleMap.get(defaultLocale)));
+				}
```

Comment:
Isn't this whole loop redundant now? Previously it accepted any non empty map, adding the default lang. as a fallback. Now we always need to add the default lang, so the loop is unnecessary; something like this  would suffice:
```
if (Validator.isNull(friendlyURLMap.get(defaultLocale)) {
    return HashMapBuilder.put(...);
}

return friendlyURLMap.
```